docker_mcp/core/backup.py

```
@@ -0,0 +1,402 @@
+        Args:
+            host: Host configuration
+            source_path: Directory path to backup
+            stack_name: Stack name for backup naming
+            backup_reason: Reason for backup (for audit trail)
+            
+        Returns:
+            Backup information dictionary
+        """
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+        backup_filename = f"backup_{stack_name}_{timestamp}.tar.gz"
+        backup_path = f"/tmp/{backup_filename}"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)